src/api/entities/Asset/types.ts

```diff
@@ -101,7 +101,7 @@ export interface AssetDetails {
   owner: Identity;
   totalSupply: BigNumber;
   fullAgents: Identity[];
-  ticker?: string;
+  ticker?: string | undefined;
```

Comment:
```suggestion
  ticker: string | undefined;
```
Other places removed the `?` (e.g. `src/api/entities/Account/types.ts`), so if that's OK then `?` and `| undefined` together seem redundant

Reply:
these might cause breaking changes - so API users then would need to explicitly state that ticker: undefined.. so I'd leave as is